springwolf-plugins/springwolf-cloud-stream-plugin/src/test/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/FunctionalChannelBeanDataTest.java

```diff
@@ -0,0 +1,107 @@
+package io.github.stavshamir.springwolf.asyncapi.scanners.channels;
+
+import org.apache.kafka.streams.kstream.KStream;
+import org.junit.Test;
+import org.springframework.context.annotation.Bean;
+
+import java.lang.reflect.Method;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+import static io.github.stavshamir.springwolf.asyncapi.scanners.channels.FunctionalChannelBeanData.BeanType.CONSUMER;
+import static io.github.stavshamir.springwolf.asyncapi.scanners.channels.FunctionalChannelBeanData.BeanType.SUPPLIER;
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+public class FunctionalChannelBeanDataTest {
+
+    @Test
+    public void testNotAFunctionalChannelBean() throws NoSuchMethodException {
+        Method method = getMethod("notAFunctionalChannelBean");
+        Set<FunctionalChannelBeanData> data = FunctionalChannelBeanData.fromMethodBean(method);
+        assertThat(data).isEmpty();
```

Comment:
style: empty lines between the given, when, then block improves readability

Reply:
Sure, will change